shoop/simple_pricing/admin_form_part.py

```diff
@@ -38,8 +38,9 @@ def _build_fields(self):
             ((shop_id or 0, group_id or 0), price)
             for (shop_id, group_id, price)
             in SimpleProductPrice.objects.filter(product=self.product)
-            .values_list("shop_id", "group_id", "price")
+            .values_list("shop_id", "group_id", "price") if group_id
```

Comment:
Shouldn't this filtering be done in a `.filter()`/`.exclude()` clause?

Also, should a migration perhaps clean groupless `SimpleProductPrice`s out so this won't be necessary?